src/osd/scrubber_common.h

```diff
@@ -49,11 +49,14 @@ enum class scrub_prio_t : bool { low_priority = false, high_priority = true };
 using act_token_t = uint32_t;
 
 /// "environment" preconditions affecting which PGs are eligible for scrubbing
+/// (note: struct size should be kept small, as it is copied around)
```

Comment:
Remove the above static assert or make sure it's within a few lines of the struct and this explanation.

Reply:
fixed